docs/document/content/user-manual/shardingsphere-jdbc/spring-boot-starter/data-source.en.md

```diff
@@ -9,6 +9,7 @@ chapter = true
 ### Configuration Item Explanation
 
 ```properties
+spring.shardingsphere.enabled= # Whether enable shardingsphere or not, default value: true
```

Comment:
Does the configuration come from ShardingSphere?

Reply:
I think should be remove this about this configuration :

```
    {
      "name": "spring.shardingsphere.enabled",
      "type": "java.lang.Boolean",
      "description": "Whether enable shardingsphere or not.",
      "defaultValue": true
    }
```

It 's from additional-spring-configuration-metadata.json in `shardingsphere-jdbc-core-spring-boot-starter` module.

Reply:
> I think should be remove this about this configuration :
> 
> ```
>     {
>       "name": "spring.shardingsphere.enabled",
>       "type": "java.lang.Boolean",
>       "description": "Whether enable shardingsphere or not.",
>       "defaultValue": true
>     }
> ```
> 
> It 's from additional-spring-configuration-metadata.json in `shardingsphere-jdbc-core-spring-boot-starter` module.

After testing, it seems that it is a feature of springboot. If `ShardingSphere` do not provide configuration, it can still take effect. So, i think we can remove it?

Reply:
It is recommended to keep this configuration

Reply:
It can flexibly turn on or off the use of shardingsphere